builder-support/debian/dnsdist/debian-buster/dnsdist.postinst

```diff
@@ -18,6 +18,12 @@ case "$1" in
 
     adduser --force-badname --system --home /nonexistent --group \
         --no-create-home --quiet _dnsdist || true
+
+    if [ "`stat -c '%U:%G' /etc/powerdns/dnsdist.conf`" = "root:root" ]; then
+      chown root:_dnsdist /etc/powerdns/dnsdist.conf
```

Comment:
it would appear that your packages are - as debians packages - configured with `--sysconfdir=/etc/dnsdist`, so this should have been `/etc/dnsdist/dnsdist.conf` ...